packages/icon/src/Icon.story.tsx

```diff
@@ -87,3 +87,5 @@ export const AllIcons: ComponentStory<typeof Icon> = (
     })}
   </div>
 );
+
+export const Error = () => <Icon glyph="error" />;
```

Comment:
nit: I understand the thought behind `glyph="error"`, but that kinda sounds like a pretty realistic name. Could use something like `this-should-cause-an-error` or something dumb like that